src/main/java/com/github/bordertech/utils/Symbols.java

```diff
@@ -32,7 +38,34 @@
 			{"\u2039", "<"},
 			{"\u203A", ">"},
 			{"\u02DC", "~"},
-			{"\u00A0", " "}
+			{"\u0020", " "},
+			{"\u00A0", " "},
+			{"\u1680", " "},
+			{"\u180E", " "},
+			{"\u2000", " "},
+			{"\u2001", " "},
+			{"\u2002", " "},
+			{"\u2003", " "},
+			{"\u2004", " "},
+			{"\u2005", " "},
+			{"\u2006", " "},
+			{"\u2007", " "},
+			{"\u2008", " "},
+			{"\u2009", " "},
+			{"\u200A", " "},
+			{"\u200B", " "},
```

Comment:
I think this should probably translate to nothing (an empty string) - see my comments on "Zero Width Space"

Reply:
Yes, I have read your comments and I will make this change.